cmd/receptor.go

```diff
@@ -23,22 +23,33 @@ type nodeCfg struct {
 	DataDir      string `description:"Directory in which to store node data"`
 }
 
-func (cfg nodeCfg) Prepare() error {
+func (cfg nodeCfg) Init() error {
```

Comment:
The point of this change is to make sure that this function, which instantiates all the MainInstance global variables, gets called _first_, before any Prepare() functions that might try to make use of these variables.